Dockerfile

```diff
@@ -1,7 +1,7 @@
 ARG ARCH
 ARG GO_VERSION
 
-FROM golang:${GO_VERSION} as build
+FROM golang:${GO_VERSION} AS build
```

Comment:
revert this one too - we want to keep the changes as minimal as possible because we'll need to rebase when we upgrade versions later